oauth2/handler.go

```diff
@@ -529,12 +529,12 @@ func (h *Handler) FlushHandler(w http.ResponseWriter, r *http.Request, _ httprou
 		fr.NotAfter = time.Now()
 	}
 
-	if err := h.r.OAuth2Storage().FlushInactiveAccessTokens(r.Context(), fr.NotAfter); err != nil {
+	if err := h.r.OAuth2Storage().FlushInactiveAccessTokens(r.Context(), fr.NotAfter, 1000, 100); err != nil {
```

Comment:
Here we should maybe add optional (?) params in FlushInactiveOAuth2TokensRequest to let the caller choose suitable values for the use case. However I think that these values would be safe default values for an http api, to keep request processing time small, which is always a good idea imo.

Reply:
It might be good to document that the limit should be greater than the growth rate?

Reply:
This is a general rule and we should also check this in handlers and give proper error. We might also add tests for this!

Reply:
Oh, you mean that limit should be greater than database grow rate, other than greater than batch size, now I see. Ok, we could add that also in the docs.

Reply:
👍